test/e2e/features/github.feature

```diff
@@ -0,0 +1,73 @@
+@github
```

Comment:
I don't see much change in `github.feature` and `gitlab.feature`. Any specific reason to create 2 separate files?